attribute-service/src/main/resources/configs/common/application.conf

```diff
@@ -3,11 +3,17 @@ service.name = attribute-service
 service.admin.port = 9013
 document.store {
   dataStoreType = mongo
+  appName = attribute-service-local
```

Comment:
Can we override the appName with the SERVICE_NAME environment variable (so that we can differentiate between a local run and a deployment)?

Reply:
> so that we can differentiate between a local run and a deployment

This goal is accomplished, just in a slightly different way. Instead of declaring that an env var can override this in the local conf, then in helm setting up the env var, we just directly override it in the helm conf using an existing value.

I preferred this approach for a couple reasons, nothing major though:
 - a few less lines of conf, and more standard with how other config overrides work
 - even in local runs, we often set service name as a jvm prop like below. So it seems confusing to rely on the _env var_ not being set, but the jvm prop to be set.
  ```kts
tasks.run<JavaExec> {
  jvmArgs = listOf("-Dservice.name=${project.name}")
}
```

Reply:
Makes sense